addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -56,7 +56,11 @@ const LinkPopoverWidget = Widget.extend({
         }
         const clipboard = new _ClipboardJS(
             this.$copyLink[0],
-            {text: () => this.target.href} // Absolute href
+            {text: () => {
+                let url = new URL(this.target.href);
+                url.pathname = '/@' + url.pathname;
```

Comment:
This doesn't belong in `web_editor`, this `/@` controller is a website one, meaning that this is just going to redirect you to 404 urls when website is not installed.